scripts/build-env.js

```diff
@@ -17,6 +53,10 @@ switch (process.env.BUILD_MODE) {
 }
 
 module.exports = {
-  BUILD_MODE: process.env.BUILD_MODE,
+  BUILD_MODE: buildMode,
   SDK_VERSION: sdkVersion,
+
+  // This will be injected by webpack when building the RUM synthetics bundle. When set to 'true',
+  // the bundle will skip the check to ignore init() calls during synthetics tests.
+  SYNTHETICS_BUNDLE: '',
```

Comment:
> I chose not to use a build mode (so we can build synthetics bundles with various build modes (staging / release))

do we plan to use various mode for synthetics?

Reply:
No, why?